### PR TITLE
Remove 404 page and forward to root path instead

### LINKED ISSUE #174: Design the 404 page (/src/ui/not-found.js)
Currently there is just a placeholder text. This should be a lot better.

### LINKED ISSUE #253: Proper Not-Found Page
There should be **no** lorem ipsum…

https://github.com/elan-ev/opencast-studio/blob/4661dcdc7a3e10381ddfa4009b790aba7483cca4/src/ui/not-found.js#L4-L13

### PR DESCRIPTION
**Pull request by LukasKalbertodt**
Due to a bug (`exact` missing), the 404 page was not accessible anyway.
This commit just removes the unused 404 page and explicitly forwards
to `/` (otherwise the wrong path is still in the URL bar).

Fixes #253 
Fixes #174

[Deployed here](https://test.studio.opencast.org/build-20191125110556-LukasKalbertodt-opencast-studio-000016-proper-not-found/) (though you can't test the 404 behavior as GitHub pages interferes)

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -63,12 +62,12 @@ function App({ defaultSettings = initial }) {
...
 
-              <ConnectedRoute path="/" settings={settings}>
+              <ConnectedRoute path="/" exact settings={settings}>
                 <Studio settings={settings} />
               </ConnectedRoute>
 
-              <Route path="*">
-                <NotFound />
+              <Route path="/*">
+                <Redirect to="/" />
```
Comment (lkiesow) [inline src/App.js]: Just wondering (didn't test this): Will this work with sub-paths?

**Comment by luniki** [comment]
Is this what we want? Silently redirecting?

File: src/App.js (hunk 1 above)
Comment (LukasKalbertodt) [reply src/App.js]: Yes. I just also added `exact` to the `settings` and `about` route, so `/settings/dfsdgsg` is also redirected to `/`.

**Comment by luniki** [comment]
After our last meeting we concluded that this is the expected behaviour.

**Pull request merged**